app/assets/favicons/manifest.json.erb

```diff
@@ -37,5 +37,7 @@
 			"type": "image\/png",
 			"density": "4.0"
 		}
-	]
+	],
+	"start_url": "index.html",
```

Comment:
This should probably be "/" because `https://www.openstreetmap.org/index.html` is no canonical url.